CDCgov/NEDSS-Modernization
# Added Table Component consistency for Patient Race

## Description

Table Component consistency for Patient Race

## Tickets

https://cdc-nbs.atlassian.net/browse/CNFT1-1717

## Steps to verify
1. Log in
2. Navigate to advanced search
3. Search for any patient
4. Navigate to patient profile and navigate to Events tabs and observe the Race table functioning as expected.

## Checklist before requesting a review
